dashboard/src/pages/TreeDetails/Tabs/LogSheet.tsx

```diff
@@ -202,7 +205,7 @@ export const LogSheet = ({
           skeletonClassname="h-[3rem]"
           customError={
             <div className="p-4 text-center">
-              This log url is not supported in the log viewer yet
+              <FormattedMessage id="logSheet.logQueryCustomError" />
```

Comment:
This message is for the log downloader, I don't think we should change it to a download info here; maybe change it above in the place of "Index of"